pysrc/juliacall/__init__.py

```diff
@@ -147,21 +147,32 @@ def args_from_config(config):
     CONFIG['opt_handle_signals'] = choice('handle_signals', ['yes', 'no'])[0]
     CONFIG['opt_startup_file'] = choice('startup_file', ['yes', 'no'])[0]
     CONFIG['opt_heap_size_hint'] = option('heap_size_hint')[0]
+    CONFIG['project'] = path_option('project', check_exists=True)[0]
+    CONFIG['exe'] = path_option('exe', check_exists=True)[0]
 
     # Stop if we already initialised
     if CONFIG['inited']:
         return
 
-    # we don't import this at the top level because it is not required when juliacall is
-    # loaded by PythonCall and won't be available
-    import juliapkg
+    have_exe = CONFIG['exe'] is not None
+    have_project = CONFIG['project'] is not None
+    if not have_exe and not have_project:
+        # we don't import this at the top level because it is not required when
+        # juliacall is loaded by PythonCall and won't be available, or if both
+        # `exe` and `project` are set by the user.
+        import juliapkg
 
-    # Find the Julia executable and project
-    CONFIG['exepath'] = exepath = juliapkg.executable()
-    CONFIG['project'] = project = juliapkg.project()
+        # Find the Julia executable and project
+        CONFIG['exe'] = juliapkg.executable()
+        CONFIG['project'] = juliapkg.project()
+    elif (not have_exe and have_project) or (have_exe and not have_project):
+        raise Exception("Both PYTHON_JULIACALL_PROJECT and PYTHON_JULIACALL_EXE must be set together, not only one of them.")
+
+    exe = CONFIG['exe']
```

Comment:
Please revert to using `exepath` internally, like `exepath = CONFIG['exepath']`.